.github/workflows/tox-tests.yml

```diff
@@ -64,3 +64,31 @@ jobs:
         with:
           fail_ci_if_error: true
           verbose: true
+  bandit-exitzero:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - name: Install RPM
+        run: sudo apt-get install -y rpm
+      - name: Setup Python
+        uses: actions/setup-python@v2
+        with:
+          python-version: 3.8
```

Comment:
Please pick python 3.9 for this because that's what has been used for pip-compile.

If using one python version to produce the requirements file and another to consume it, it's possible to create some scenarios where the requirements file becomes uninstallable (usually it would be something like: from python 3.9, standard library gained some feature; it can still be used in python 3.8 but you have to install an additional package for it; and since the requirements file was compiled with python 3.9 that extra package will be missing if you try to test with python 3.8).

Reply:
Thanks - that certainly sounds like a situation I'd like to avoid. Both of the Python 3.8 references have been updated to 3.9 with a force-push to this commit.